auscope-geochemistry-network/.htaccess

```diff
@@ -4,4 +4,7 @@ Options +FollowSymLinks
 RewriteEngine On 
 
 # using w3id domain: https://w3id.org/auscope-geochemistry-network/geochemistry/1.0/
-RewriteRule ^(.*)$ https://vocabs.ardc.edu.au/repository/api/lda/agn/auscope-geochemistry-network-vocabularies-for-geoc/v1-0/resource?uri=https://w3id.org/auscope-geochemistry-network/$1 [B,L,NE,QSA,R=307]
+RewriteRule ^(.*)$ https://vocabs.ardc.edu.au/repository/api/lda/agn/agn-geochemistry-vocabulary/v1-0/resource?uri=https://w3id.org/auscope-geochemistry-network/$1 [B,L,NE,QSA,R=307]
+    
+# using w3id domain: https://w3id.org/auscope-geochemistry-network/analytical-methods/1.0/
+RewriteRule ^(.*)$ https://demo.vocabs.ardc.edu.au/repository/api/lda/agn/agn-analytical-methods-vocabulary/v0-1/resource?uri=https://w3id.org/auscope-geochemistry-network/$1 [B,L,NE,QSA,R=307]
```

Comment:
The first rule will catch everything before the second is evaluated.  Based on the comments, perhaps you want to add a path in the patterns?

Reply:
Yes the second path was trying to add a redirect for a new vocabulary path. Edits to Line 7 were just to match this path to updates in the vocabulary service side